rs/embedders/src/compilation_cache.rs

```diff
@@ -29,6 +30,7 @@ const DEFAULT_MEMORY_CAPACITY: NumBytes = NumBytes::new(10 * GB);
 
 /// Stores the serialized modules of wasm code that has already been compiled so
 /// that it can be used again without recompiling.
+#[derive(DeterministicHeapBytes)]
 pub struct CompilationCache {
```

Comment:
Same here. It doesn't seem like a cache would have a deterministic size across replicas.

And for all other caches.

Reply:
LGTM, thank you.